src/yggdrasil/session.go

```diff
@@ -539,6 +576,8 @@ func (sinfo *sessionInfo) doWorker() {
 			} else {
 				return
 			}
+		case e := <-sinfo.reconfigure:
+			e <- nil
```

Comment:
There is one case where reconfiguring a session makes sense - that's if we change our TUN MTU. That's not implemented yet, but in that case we would need to update our `sessionInfo.myMTU` and then would want to send a session ping to update the remote side so that the PMTUD stuff converges quickly.